packages/terra-form-select/src/ComboboxField.jsx

```diff
@@ -66,6 +66,10 @@ const propTypes = {
    */
   // eslint-disable-next-line react/forbid-prop-types
   labelAttrs: PropTypes.object,
+  /**
+   * The max height of the dropdown.
+   */
+  maxHeight: PropTypes.number,
```

Comment:
Should this be a string prop and accept the same options as the maxWidth prop?

Reply:
The maxWidth is a propType for the [field component](https://engineering.cerner.com/terra-ui/components/terra-form-field/form-field/form-field), the select maxHeight prop only accepts a number. I'd say for these changes we should keep it consistent with the existing <Select /> maxHeight API.

https://engineering.cerner.com/terra-ui/components/terra-form-select/form-select/select

Reply:
Is max height here a style value or maximum number of options it will display?

Reply:
It's the maximum height of the dropdown interpreted as pixels and is applied as an inline style on the dropdown. It's used in height calculations so the prop required being set as a number:

https://github.com/cerner/terra-core/blob/master/packages/terra-form-select/src/shared/_FrameUtil.js#L31-L55

This prevents a user setting a custom maxHeight that is larger than the actual available height on the screen.

Reply:
ohhh gottchya

Reply:
Thanks for the extra info!